Samples/outlook-spam-reporting/README.md

```diff
@@ -39,59 +39,145 @@ To learn about key components of this sample, see [Implement an integrated spam-
+
+    ```console
+    npm install -g @microsoft/teamsapp-cli
+    ```
+
+- (Optional) If you want to deploy the sample with the unified manifest for Microsoft 365 to Microsoft Azure, you'll need the following:
+  - An Azure subscription.
+  - [Visual Studio Code](https://code.visualstudio.com/).
+  - [Teams Toolkit extension for Visual Studio Code](https://learn.microsoft.com/microsoftteams/platform/toolkit/install-teams-toolkit).
 
 ## Run the sample
 
-Run this sample in Outlook on Windows (new or classic) or on the web using one of the following add-in file hosting options.
+Run this sample with a [unified manifest for Microsoft 365](#run-with-the-unified-manifest-for-microsoft-365) or [add-in only manifest](#run-with-the-add-in-only-manifest). Use one of the following add-in file hosting options.
 
-### Run the sample from GitHub
+> [!NOTE]
+>
+> - The spam-reporting sample with the unified manifest for Microsoft 365 is currently only supported in classic Outlook on Windows. To run the sample in Outlook on the web or the new Outlook on Windows, use the add-in only manifest.
+>
+> - To learn more about manifests, see [Office Add-ins manifest](https://learn.microsoft.com/office/dev/add-ins/develop/add-in-manifests).
 
-1. Download the **manifest.xml** file from this sample to a folder on your computer.
-1. Sideload the add-in manifest in Outlook on Windows (new or classic) or on the web by following the manual instructions in [Sideload Outlook add-ins for testing](https://learn.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing#sideload-manually).
-1. Follow the steps in [Try it out](#try-it-out) to test the sample.
+### Run with the unified manifest for Microsoft 365
 
-### Run the sample from localhost
+#### Use GitHub as the web host
 
-If you prefer to host the web server for the sample on your computer, follow these steps.
+The quickest way to run the sample is to use GitHub as the web host. However, you can't debug or change the source code. The add-in web files are served from this GitHub repository.
 
-1. Install a recent version of [npm](https://www.npmjs.com/get-npm) and [Node.js](https://nodejs.org/) on your computer. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
-1. You need http-server to run the local web server. If you haven't installed this yet, run the following command.
+1. Download the **outlook-spam-reporting.zip** file from this sample to a folder on your computer.
+1. Sideload the sample to Outlook by following the instructions in [Sideload with the Teams Toolkit CLI (command-line interface)](https://learn.microsoft.com/office/dev/add-ins/testing/sideload-add-in-with-unified-manifest#sideload-with-the-teams-toolkit-cli-command-line-interface).
+1. Follow the steps in [Try it out](#try-it-out) to test the sample.
+
```

Comment:
Should we tell readers how to stop and uninstall with `teamsapp uninstall`? In the localhost section, we tell them how to stop.

Reply:
Added.